charts/ocis/docs/values.adoc.yaml

```diff
@@ -186,6 +186,10 @@ features:
       # -- Reference to a ConfigMap containing images that can be referenced from the html mail template.
       # This ConfigMap is optional and can be omitted when images are not used.
       htmlMailImagesConfigRef: "html-mail-images"
+
+  # -- Disables server-sent events (sse). When disabled, clients will no longer receive sse notifications.
+  disableSse: false
```

Comment:
I'd prefere this to be in a sse subsection like:

```
features:
  sse:
    disabled: false
```

This makes it future proof regarding more sse settings